src/gluonts/model/seq2seq/_forking_estimator.py

```diff
@@ -363,51 +380,92 @@ def create_transformation(self) -> Transformation:
         return Chain(chain)
 
     def create_training_network(self) -> ForkingSeq2SeqNetworkBase:
-        return ForkingSeq2SeqTrainingNetwork(
-            encoder=self.encoder,
-            enc2dec=FutureFeatIntegratorEnc2Dec(),
-            decoder=self.decoder,
-            quantile_output=self.quantile_output,
-            context_length=self.context_length,
-            cardinality=self.cardinality,
-            embedding_dimension=self.embedding_dimension,
-            scaling=self.scaling,
-            scaling_decoder_dynamic_feature=self.scaling_decoder_dynamic_feature,
-            dtype=self.dtype,
-        )
+        if self.sampling:
+            return ForkingSeq2SeqTrainingNetwork(
+                encoder=self.encoder,
+                enc2dec=FutureFeatIntegratorEnc2Dec(),
+                decoder=self.decoder,
+                distr_output=self.distr_output,
```

Comment:
The sampling should not yield quantile output by default. The sampling option should yield samples from the specified distribution. We possible need another if else to cover the quantile output option.